tools/fetch_network_harness.js

```diff
@@ -0,0 +1,41 @@
+const dgram = require('node:dgram');
+const server = dgram.createSocket('udp4');
+const fs = require('node:fs');
+const path = require('node:path');
+
+//  qemu-system-i386 -m 2G -nographic -hda ~/disk.qcow2 -netdev dgram,id=net0,remote.type=inet,remote.host=127.0.0.1,remote.port=6677,local.host=127.0.0.1,local.port=7744,local.type=inet -device e1000,netdev=net0
+
+globalThis.dbg_assert = require('node:assert');
+
+let FetchNetworkAdapter = eval(fs.readFileSync(path.join(__dirname, 'browser', 'fetch_network.js'), 'utf-8') + ";FetchNetworkAdapter");
```

Comment:
At the bottom fetch_network.js, you could write:

```js
if(typeof module !== "undefined" && typeof module.exports !== "undefined")
{
    module.exports["FetchNetworkAdapter"] = FetchNetworkAdapter;
}
```

and use `require` instead.